### ISSUE TITLE
Movie frame data type inconsistent between operating systems

### ISSUE THREAD

**Issue by bendichter** [inscopix/py_isx#48]
For reading `"movie_u8.isxd"` and `"movie_longer_than_3_min.isxd"` on unix operating systems, `Movie.read(file_path).get_frame_data(0).dtype` results in `dtype('int64')` , however on Windows, the dtype is `dtype('int32')`. Is this the intended behavior?

**Comment by bendichter** [comment]
A potentially bigger issue, 

```ipython
In [7]: Movie.read(file_path).get_frame_data(0).dtype
Out[7]: dtype('int64')

In [8]: Movie.read(file_path).data_type
Out[8]: numpy.uint8
```

**Comment by sg-s** [comment]
hmmm, interesting. i don't have a windows computer so that's a good catch. definitely a bug IMO

**Comment by brunoboivin** [comment]
@bendichter Addressed in PR #52 . The inconsistency observed on Windows is due to numpy's default integer type being platform-dependent and not enforced in the `py_isx` package when reading movie data.

**Comment by brunoboivin** [comment]
@bendichter Fix released as part of version 1.0.4. To use the new version:
```
pip install isx==1.0.4
```
